presto-pinot-toolkit/src/test/java/com/facebook/presto/pinot/query/TestPinotQueryGenerator.java

```diff
@@ -324,6 +367,25 @@ public void testAggregationWithOrderByPushDownInTopN()
         TableScanNode tableScanNode = tableScan(planBuilder, pinotTable, city, fare);
         AggregationNode agg = planBuilder.aggregation(aggBuilder -> aggBuilder.source(tableScanNode).singleGroupingSet(variable("city")).addAggregation(planBuilder.variable("agg"), getRowExpression("sum(fare)", defaultSessionHolder)));
         TopNNode topN = new TopNNode(planBuilder.getIdAllocator().getNextId(), agg, 50L, new OrderingScheme(ImmutableList.of(new Ordering(variable("city"), SortOrder.DESC_NULLS_FIRST))), TopNNode.Step.FINAL);
-        testPQL(pinotConfig, topN, "", defaultSessionHolder, ImmutableMap.of());
+        testPinotQuery(pinotConfig, topN, "", defaultSessionHolder, ImmutableMap.of());
+    }
+
+    @Test
```

Comment:
If this test is only for the SQL end point, should it go in `TestPinotQueryGeneratorSql` ?

Reply:
This should work for both pql and sql

Reply:
done.